aiida_vasp/commands/mock_vasp.py

```diff
@@ -49,6 +49,7 @@ def mock_vasp():
 
     system = incar_parser.get_quantity('incar', {})['incar'].get_dict().get('system', '')
     test_case = re.findall(r'test-case:(.*)$', system)
+    print("CASE", test_case)
```

Comment:
use `click.echo()` for CLI output, since it is already py2/py3 compatible. also it can handle printing to stderr / colored output etc.

Reply:
This is just from testing. Removed.